Massonus/HW5/HW5.1.py

```diff
@@ -0,0 +1,13 @@
+numbers = []
+while True:
+    user_input = input("Enter a number, or if you want to quit enter 'q': ")
+    if user_input == 'q':
+        break
+    numbers.append(user_input)
+
+print("Your integer numbers are: ", numbers)
+
+for number in numbers:
```

Comment:
good idea
look at this version
list_of_number = [2, 1, 4, 6, 1, 4, 8, 6]

for i in range(len(list_of_number)):
    list_of_number[i] = float(list_of_number[i])
print(list_of_number)